ddl/db_partition_test.go

```diff
@@ -1952,6 +1953,88 @@ func TestDropPartitionWithGlobalIndex(t *testing.T) {
 	require.Equal(t, 2, cnt)
 }
 
+func TestGlobalIndexReaderInDropPartition(t *testing.T) {
+	t.Skip("not support yet")
```

Comment:
`partitionIds` for global index will be pushed down in a graceful manner in future.